src/main/java/com/craftingguide/exporter/extensions/craftingguide/ItemIconDumperScreen.java

```diff
@@ -304,6 +310,7 @@ private void drawItem(ItemModel item, int i, int j) {
                 DRAW_ITEMS.renderItemAndEffectIntoGUI(fontRenderer, renderEngine, itemStack, i, j);
             } else {
                 DRAW_ITEMS.renderItemIntoGUI(fontRenderer, renderEngine, itemStack, i, j);
+                DRAW_ITEMS.renderItemOverlayIntoGUI(fontRenderer, renderEngine, itemStack, i, i);
```

Comment:
Does this actually work for you?  I had many cases where this screwed up the images for things with a "magical" effect (e.g., magic apples, potions, etc.).

Reply:
works for me.